build.py

```diff
@@ -261,26 +254,16 @@ def build_package(target, build_native_package):
 
     # get
     if target == 'clang':
```

Comment:
maybe clang doesn't have to be different now?

since it's just a single repository, like all the others?

i like the 'clone only the version we want'... that might be good/best for the default behavior for all the externals...

Reply:
maybe we capture this in an issue for later as well.

Reply:
agreed